setup.py

```diff
@@ -27,7 +27,7 @@
     "djangorestframework>=3.11.1,<4.0",
     "django-filter>=2.2,<23",
     "draftjs_exporter>=2.1.5,<3.0",
-    "Pillow>=4.0.0,<10.0.0",
+    "Pillow>=4.0.0,<11.0.0",
```

Comment:
```suggestion
    "Pillow>=9.1.0,<11.0.0",
```

As https://pillow.readthedocs.io/en/stable/releasenotes/index.html mentions

> Please use the latest version of Pillow. Functionality and security fixes should not be expected to be backported to earlier versions.